test/testdata/ruby_benchmark/stripe/hash_transform_values.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+# typed: true
+# compiled: true
+
+i = 0
+
+# 8 is a tipping point on the performance of this benchmark. Initially the compiler is about 2x faster than the
+# interpreter, but anything above 8 starts to stretch out the interpreted time substantially, and really show the
+# benefit of having compiled blocks.
```

Comment:
Presumably this is because 8 is the tipping point for array vs. hash storage in hash tables?